julia/caesar-api/icra-1-simple.ipynb

```diff
@@ -3878,7 +3878,7 @@
     "\n",
     "The figure shows the position and orientation (red forward) for poses `x0` and `x1`. As well as the covariance ellipse. \n",
     "Since the solver used was non-parametric, the covariance ellipse is based on a best Gaussian distribution fit of the full belief.\n",
-    "A few other functions are also handy for interacting with the factor graph, for instance `getVariable` returns the full variable.\n",
+    "A few other functions are also handy for interacting with the factor graph, for instance, `getVariable` returns the full variable.\n",
```

Comment:
I don't think there's a second comma here actually

Reply:
ok, I removed it